src/Accordion/accordion-style.scss

```diff
@@ -1,6 +1,5 @@
-$blue-dark: #0092e1;
-$white: #fff;
-$gray-8: #e6e6e6 !default;
+@import "../../node_modules/@sebgroup/bootstrap/scss/variables";
```

Comment:
There most be a way to create alias for variables file
https://medium.com/front-end-weekly/creating-alias-for-package-imports-in-react-99d455284029